tools/pylib/boutdata/squashoutput.py

```diff
@@ -70,21 +66,37 @@ def squashoutput(datadir=".", outputname="BOUT.dmp.nc", format="NETCDF4", tind=N
         Append to existing squashed file
     delete : bool
         Delete the original files after squashing.
+    progress : bool
+        Print a progress bar
+    docontinue : bool
+        Try to progress a previously interrupted squash
     """
 
+    from boutdata.data import BoutOutputs
+    from boututils.datafile import DataFile
+    from boututils.boutarray import BoutArray
+    from zoidberg import progress as bar
+    import numpy
+    import os
+    import tempfile
+    import shutil
+    import glob
+
     fullpath = os.path.join(datadir,outputname)
 
     if append:
-        import tempfile
-        import shutil
-        import glob
         datadirnew = tempfile.mkdtemp(dir=datadir)
         for f in glob.glob(datadir+"/BOUT.dmp.*.??"):
             if not quiet:
                 print("moving",f)
             shutil.move(f,datadirnew)
         oldfile=datadirnew+"/"+outputname
         datadir=datadirnew
+    if docontinue:
+        if append:
+            raise NotImplemented("append & docontinue: Case not handled")
```

Comment:
Typo, should be `NotImplementedError` -- `NotImplemented` is a special value for returning from arithmetic operators